spec/workers/link_check_worker_spec.rb

```diff
@@ -95,14 +97,16 @@
           end
         end
 
-        context 'exceptions' do
+        context 'when there is an exception' do
           it 'handles throttling error' do
```

Comment:
Is this testing exceptions are swallowed? If so could it be `it 'swallows ThrottleLimitError'`

Reply:
Fair point

Reply:
Done!